src/nvim/window.c

```diff
@@ -108,7 +108,7 @@ do_window (
      * don't replicate the quickfix buffer. */
     if (bt_quickfix(curbuf))
       goto newwindow;
-    win_split((int)Prenum, WSP_VERT);
+    (void)win_split((int)Prenum, WSP_VERT);
```

Comment:
Those 2 are executing user commands that split windows. One might want to return an error to the user if the split fails, but apart from that, there's probably not much to do.